Orchestrator.py

```diff
@@ -290,12 +290,14 @@ def __init__(self, numDotBots, initialPosition):
         self.dotbotsview     = [
             {
                 # evaluated position of the DotBot when it last stopped
-                'x':              x,
-                'y':              y,
+                'x':                  x,
+                'y':                  y,
                 # current movement
-                'heading':        0,
-                'speed':          0,
-                'movementSeqNum': 0,
+                'heading':            0,
+                'speed':              0,
+                'movementSeqNum':     0,
```

Comment:
to use a common prefix, please call these elements `seqNumMovement` and `seqNumNotification`